lib/IRGen/GenClangType.cpp

```diff
@@ -315,6 +315,14 @@ ClangTypeConverter::reverseBuiltinTypeMapping(IRGenModule &IGM,
   cacheStdlibType(#SWIFT_TYPE_NAME, clang::BuiltinType::CLANG_BUILTIN_KIND);
 #include "swift/ClangImporter/BuiltinMappedTypes.def"
 
+  // On 64-bit Windows, no C type is imported as an Int or UInt; CLong is
+  // imported as an Int32 and CLongLong as an Int64. Therefore, manually
+  // add mappings to C for Int and UInt.
+  if (IGM.Triple.isOSWindows() && IGM.Triple.isArch64Bit()) {
+    cacheStdlibType("UInt", clang::BuiltinType::ULongLong);
+    cacheStdlibType("Int", clang::BuiltinType::LongLong);
```

Comment:
`[u]intptr_t` might be a better choice here.

Reply:
I can't find a definition for pointer types in Clang's `BuiltinTypes.def`. Where would that be defined, or is there a different way we could achieve that mapping?

Reply:
You can get Clang's choice for (u)intptr_t from clang::ASTContext::get(U)IntPtrType. There'll be a builtin type under that.